onnxruntime/test/onnx/main.cc

```diff
@@ -312,8 +312,6 @@ int real_main(int argc, char* argv[]) {
       {"scatter_without_axis", "opset 9 not supported yet"},
       {"scan_sum", "opset 9 not supported yet"},
       {"shrink", "opset 9 not supported yet"},
```

Comment:
Keeping "shrink" test excluded because of bug in test input - https://github.com/onnx/onnx/issues/1823

Reply:
Can you create a task to fix this when [https://github.com/Microsoft/onnxruntime/pull/499](https://github.com/Microsoft/onnxruntime/pull/499) gets checked in to update the ONNX version?

---
In reply to: [258262246](https://github.com/Microsoft/onnxruntime/pull/485#discussion_r258262246) [](ancestors = 258262246)